### PR TITLE
Component

### PR DESCRIPTION
**Pull request by elarhadu**
In this milestone, I;
- Created a directory called components.
- Inside components,I created a new Calculator.js file.
- In the Calculator.js,I created a React component that matches the design in the project requirement

### CODE REVIEW

File: src/components/calculator.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+/* eslint-disable react/no-array-index-key */
+/* eslint-disable react/prefer-stateless-function */
```
Comment (DammyShittu) [inline src/components/calculator.js]: - [ ] Kindly remove these `eslint-disable` comments as adding them defeats the purpose of linters. Fix the errors instead.

  The first one seems to be unnecessary as when I removed it eslint throws no errors. For the second one you need to add a state to the component.

  ```
  export default class Calculator extends Component {
    constructor(props) {
      super(props);
  
      this.state = {};
    }
  ...
  ```

File: src/App.js [hunk 2]
```diff
@@ -1,29 +1,14 @@
-import logo from './logo.svg';
-import './App.css';
+/* eslint-disable react/prefer-stateless-function */
+import React, { Component } from 'react';
+// eslint-disable-next-line import/extensions
+import Calculator from './components/Calculator.js';
```
Comment (DammyShittu) [inline src/App.js]: - [ ] Kindly remove eslint disabling here also.

  For the first one use suggestion above. For the second one you need to define the import correctly to match the file name (small letter "c", not the capital letter "C"). You may get an extension error - just remove `.js`.

  ```
  import Calculator from './components/calculator';
  ```

File: src/components/calculator.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+import React, { Component } from 'react';
+import './calculator.stlye.css';
```
Comment (DammyShittu) [inline src/components/calculator.js]: - **[OPTIONAL]** I think you meant to name this file `style`, not `stlye`

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @elarhadu,

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [ ] Kindly remove these files.

  urr

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mwanawabangona — APPROVED** [review]
Hi @elarhadu,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations!  


## Highlights
*   No linter errors :heavy_check_mark:
*   Repo looks professional :briefcase:
*   Solid pull request scroll
*   Good quality commits
*   Good job implementing the requested changes!! :clap: :clap: :clap:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**